bosh-stemcell/lib/bosh/stemcell/archive_filename.rb

```diff
@@ -24,7 +24,12 @@ def to_s
     private
 
     def name
-      definition.light? ? "light-#{base_name}" : base_name
+      mod_name = definition.light? ? "light-#{base_name}" : base_name
+      if RbConfig::CONFIG['host_cpu'] == 'powerpc64le'
```

Comment:
Can we pull this (`RbConfig::CONFIG['host_cpu'] == 'powerpc64le'`) out into some constant?